backend/municipal-services/ingestion-service/app/api/endpoints/template_generation.py

```
@@ -113,3 +117,89 @@ async def get_facility_ingestion_template_with_supervisors(
         cleanup_temp_file(output_file_path)
         raise HTTPException(status_code=500, detail=f"An unexpected error occurred: {str(e)}")
 
+@router.get('/facilitySelection',
+            summary='Generate facility selection template Excel file',
+            response_description="Returns Excel template with facility data")
+async def get_facility_selection_template(
+        facility_service: FacilityTemplateService = Depends(),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)